_test/test_instrument_classes/test_IX_detectors_3.m

```diff
@@ -0,0 +1,122 @@
+classdef test_IX_detectors_3 < TestCaseWithSave
+    % Test the calculation of quantities for a detector object
+    properties
+        dia
+        height
+        thick
+        atms
+        path
+    end
+    
+    methods
+        %--------------------------------------------------------------------------
+        function self = test_IX_detectors_3 (name)
+            self@TestCaseWithSave(name);
+            
+            % Arrays for construction of detectors
+            dia(1) = 0.0254;  height(1) = 0.015; thick(1) = 6.35e-4; atms(1) = 10; th(1) = pi/2;
+            dia(2) = 0.0300;  height(2) = 0.025; thick(2) = 10.0e-4; atms(2) = 6;  th(2) = 0.9;
+            dia(3) = 0.0400;  height(3) = 0.035; thick(3) = 15.0e-4; atms(3) = 4;  th(3) = 0.775;
+            dia(4) = 0.0400;  height(4) = 0.035; thick(4) = 15.0e-4; atms(4) = 7;  th(4) = 0.775;
+            dia(5) = 0.0400;  height(5) = 0.035; thick(5) = 15.0e-4; atms(5) = 9;  th(5) = 0.775;
+            
+            self.dia = dia;
+            self.height = height;self;
```

Comment:
Why is `self;` here? Debugging?

Reply:
storing in the test object for use in the tests